### ISSUE TITLE
Possible additions

### ISSUE THREAD

**Issue by dkasy64** [NicholasTourony/qBay#17]
We could maybe add more categories for items

**Comment by NicholasTourony** [comment]
I agree that there should be more categories for items since college students might have a lot to sell.
